packages/flutter_tools/test/general.shard/web/chrome_test.dart

```diff
@@ -525,6 +525,38 @@ void main() {
       throwsToolExit(message: 'Failed to launch browser.'),
     );
```

Comment:
@jonahwilliams btw, in my experiments the last test (if I disable the newly added one) is passing as well if I replace `throwsToolExit` by `returnsNormally`. I believe the last test just does not have time to finish since it is not awaited - should this and other tests use `await expectLater` or `await expectToolExitLater` instead?

Reply:
that sounds reasonable to me